repldex/backend/templates/index.html

```diff
@@ -141,7 +141,10 @@ <h1 class="title">Repldex</h1>
 			{% endif %}
     {% endif %}
   </div>
-	<input id="search-bar" placeholder="Search entries">
+    <label>
+        Search entries
```

Comment:
This should be an aria label or something so it doesn't show the text normally

Reply:
Sure